src/hooks/sdk/docs/useSelectVideoInputDevice.stories.mdx

```diff
@@ -51,3 +56,50 @@ const MyChild = () => {
 ### Dependencies
 
 - `AudioVideoProvider`
+
+
+### Why does my Webcam LED light turns on when I change a video input device using `selectVideo` from `useSelectVideoInputDevice` hook but no local video starts and the LED stays `ON`?
+
+Whenever you change and select a video input device using the `useSelectVideoInputDevice` hook, it internally calls the `chooseVideoInputDevice` API from the Amazon Chime SDK for JavaScript.
+The `chooseVideoInputDevice` API internally calls `getUserMedia` with requested video constraints.
+When successful, it triggers the LED light to go `ON`. Hence, we suggest to use the hook to select the video input device when you want to actually start the local video.
```

Comment:
Do you mean the LED light is `ON`, but no video actually starts?

Reply:
Have updated the documentation, so removed this.